### PR TITLE
fix(config): Added Gruntfile linting

### LINKED ISSUE #1125: Gruntfile not being linted
Hi there,

As it stands, the Gruntfile makes use of the Lodash `union` method to specify the ESLint `target`:

`_.union(defaultAssets.server.gruntConfig, defaultAssets.server.allJS, defaultAssets.client.js, testAssets.tests.server, testAssets.tests.client, testAssets.tests.e2e)`

This method only operates on arrays, however, and so the `gruntConfig` property in the default assets  (a string) isn't being added to this union.

May I suggest changing the field in `config/assets/default.js` from `gruntConfig: 'Gruntfile.js'` to `gruntConfig: ['Gruntfile.js']` so that it may be linted as expected?

Happy to submit a PR!

### PR DESCRIPTION
**Pull request by JulianLaval**
Changed config properties to arrays for inclusion in Gruntfile Lodash unions, e.g. linting. Also fixes typo in CONTRIBUTING.md

Fixes #1125

### CODE REVIEW

**Comment by codydaig** [comment]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Fixes #82
   * The scope should be where the change took place.
   * Examples: users, core, config, articles
 * Subject:
-  * The subject line should be clear and consice as to what is being accomplished in the commit.
+  * The subject line should be clear and concise as to what is being accomplished in the commit.
```
Comment (ilanbiala) [inline CONTRIBUTING.md]: Different PR just because it's separate from the purpose of this one.

**Comment by ilanbiala** [comment]
LGTM other than the typo, which should probably be in a separate PR because it fixes something different.

**Comment by JulianLaval** [comment]
Fair enough! Assumed that since the change was minor, it could be bundled with this PR - will go ahead and update this one / create a new one for the typo.

**Pull request merged**